miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/userinfo/Contact.kt

```diff
@@ -7,5 +7,6 @@ import androidx.annotation.Keep
 data class Contact(
     val id: String,
     var name: String?,
-    var email: String?
+    var email: String?,
+    var allEmailList: List<String> = emptyList()
```

Comment:
why empty empty list and not `List<String>?`

Reply:
@Climbatize in that way hostapp it's optional, so host app can ignore it, is it a mandatory field?

Reply:
got it! Could have been `var allEmailList: List<String>? = null` then